lib/views/profile.js

```diff
@@ -48,13 +53,18 @@ function display (req, res) {
     return
   }
 
-  const locals = {
-    config: config.get(),
-    section: 'profile',
+  var locals = {
     user,
     externalAuthProvider: dotget(config.get('externalAuth'), 'provider'),
     externalProfiles
   }
-
-  res.send(pug.renderFile('templates/views/profile.jade', locals))
+  if (!req.accepts('text/html')) {
+    return res.status(200).header('content-type', 'application/json').send(locals)
```

Comment:
This locals is okay

Reply:
This is sending back the user and any external (OAuth) information.  Are you think I should send only back a subset?  I was curious about this.  It does send back the encrypted password currently.  Is that a concern?

Reply:
Hmm, doesn't sound great to send back, but I don't know off the top of my head